apps/src/netsim/NetSimRouterTab.js

```diff
@@ -21,12 +21,14 @@ var NetSimRouterStatsTable = require('./NetSimRouterStatsTable');
  * Generator and controller for router information view.
  * @param {jQuery} rootDiv - Parent element for this component.
  * @param {netsimLevelConfiguration} levelConfig
- * @param {function} bandwidthChangeCallback
- * @param {function} memoryChangeCallback
+ * @param {Object} callbacks
+ * @param {function} callbacks.bandwidthSliderChangeCallback
```

Comment:
Hmm... that's true, technically neither of these callbacks is required down at the `NetSimSlider` level.  That said, I would be a little concerned if a `NetSimRouterTab` was constructed without these callbacks, even though I don't require them to exist in code.  I figure my documentation should reflect the intended use of the class, not just its minimum "non-crashing" requirements.